### ISSUE TITLE
Mitmproxy problem with HTTPS

### ISSUE THREAD

**Issue by MuhammadBor** [mitmproxy/mitmproxy#3283]
I have setup mitmproxy properly.
I use arpspoof attack correctly on my nettwork so as to perform MITM attack. the “arp -a” shows that arpsoof works appropriately.
I use the following command before starting mitmproxy process.

    sysctl -w net.ipv4.ip_forward=1
    iptables -t nat -A PREROUTING -i wlan0 -p tcp --dport 80 -j REDIRECT --to-port 8080
    iptables -t nat -A PREROUTING -i wlan0 -p tcp --dport 443 -j REDIRECT --to-port 8080

Then, i install certificate on my device which is android and name it “mitmproxy-ca-cert”, and it sounds it is installed in correct way.

Then i start mitmproxy using:

    mitmproxy --mode transparent --showhost

The mitmproxy works great on HTTP request, but any request with HTTPS, the page could not be load on victim browser and nothing shown in mitmproxy terminal.

Any suggestion to resolve that? is there any command i may add to solve this problem.

Note: Attacker Os: Kali Linux 2019
Victim Os: Android 8

**Comment by Prinzhorn** [comment]
I doubt this is still relevant after four years. Feel free to open a new issue if you are running into trouble with the latest mitmproxy version.
